datafusion/src/lib.rs

```diff
@@ -230,6 +230,9 @@ pub mod variable;
 pub use arrow;
 pub use parquet;
 
+#[cfg(feature = "pyarrow")]
+pub mod pyarrow;
```

Comment:
Does the whole module really need to be public?

Reply:
I don't think so, will update.